stdlib/doc_for_pervasives

```diff
@@ -0,0 +1,2 @@
+[@@@deprecated "Use Stdlib"]
+include module type of struct include Stdlib end
```

Comment:
A couple of thoughts:

- moving forward, we probably won't keep pervasive.ml up to date, i.e. new functions will only go in stdlib.mli. That means that the manual will show a few functions that don't exist
- transiently creating pervasives.mli could create a race condition in the build

What about systematically creating pervasives.mli with something like this?

```makefile
pervasives.mli: pervasives.ml
        echo '[@@@deprecated "Use Stdlib instead."]' > $@
        $(CAMLC) -nostdlib -I . -i $< >> $@
```

Reply:
Creating pervasives.mli sounds like a better idea indeed.

Reply:
I thought of an easier solution: just use `pervasives.ml` directly after adding a deprecated doc comments.